sqrl-tools/sqrl-cli/Dockerfile

```diff
@@ -1,15 +1,25 @@
 FROM eclipse-temurin:11
 WORKDIR /usr/src/app
+ARG POSTGRES_VERSION=14
```

Comment:
I see version 14 here but version 16 down below. Is this intended?

Reply:
Also, the arg isn't required, we should just fix to the version that the container installs

Reply:
thanks for the message. I fixed the comment. I installed the same version as postgres in docker container. 

As for the ARG, I try to parameterize the version. later on also use that arg probably for postgres installation. How do you think about that ?

Reply:
so that pgvector version always match with postgresql installation

Reply:
Instead of downgrading to 14, can we use postgres 15 or higher? Our test coverage is around pg 15.

Reply:
Yeah, it'd be nice to have the pgvector version match the pg version.

Reply:
ok, will try to upgrade to v15

Reply:
done